assets/js/modules/adsense/components/setup/v2/SetupAccount.js

```diff
@@ -109,9 +109,9 @@ export default function SetupAccount( { account } ) {
 	}
 
 	if (
-		state === SITE_STATE_NEEDS_ATTENTION ||
-		state === SITE_STATE_REQUIRES_REVIEW ||
-		state === SITE_STATE_GETTING_READY
+		accountState === STATE_NEEDS_ATTENTION ||
+		site?.state === STATE_REQUIRES_REVIEW ||
+		site?.state === STATE_GETTING_READY
```

Comment:
These look like new constants specific to account state: https://github.com/google/site-kit-wp/pull/5109/files#diff-7a4be23cb854366543b75e18911a77d95f74ca436570d142fff54c6b20d8e429R29-R31

But they're used for site state. Why create the new constants?

Reply:
See my above comment, this is incorrect looking at the `site`, it needs to look at the client for the last two states here.